examples/plugin/plugin.toml

```diff
@@ -0,0 +1,28 @@
+[shared]
+gatewaysvc = "localhost:19000"
+plugin = true
+
+# [registry]
+# driver = "static"
+#
+# [registry.static]
+# services = ["authprovider","userprovider"]
+#
+# [registry.static.authprovider]
+# bearer = ["localhost:0123"]
+# basic = ["localhost:1234"]
+# publiclink = ["localhost:9876"]
+
+[grpc]
+address = "0.0.0.0:19000"
+
+[grpc.services.gateway]
+userprovidersvc = "localhost:19000"
+
+
+[grpc.services.userprovider]
+# replace the driver with the plugin binary file
+driver = "/path/to/binary"
```

Comment:
Currently, I just have this dummy path. Should I also provide the actual binary for the purpose of testing?

Reply:
Yep, the full path so is clear to the reader.

Reply:
Okay. I'll add the plugin binary in the example folder.